Resources/ArmTemplates/template.json

```diff
@@ -8,6 +8,12 @@
                 "description": "The APIM proxy address including the protocol and trailing slash. eg https://dev.api.nationalcareersservice.org.uk/"
             }
         },
+        "ApimDysacKey": {
```

Comment:
The name of this parameter, and it's type indicates it's a secret, yet the metadata indicates it's a url.
Which is it?